glossary.yml

```diff
@@ -456,6 +490,14 @@
       A specification for a [build manager](#build_manager) that describes how
       some files depend on others and what to do if those files are out of date.
 
+- slug: build_stale
```

Comment:
Should the slug be `stale_build` since the term is "stale (in build)"? This would follow the convention of [`child_tree`](https://glosario.carpentries.org/en/#child_tree) referring to a "child (in a tree)".

Reply:
All of the other terms related to builds have `build_` as a prefix, so I stuck with that. We should probably discuss/standardize - I could see `child_tree` being `tree_child`.